vendor/vendor.json

```diff
@@ -55,10 +55,10 @@
 			"revisionTime": "2015-02-12T10:17:44Z"
 		},
 		{
-			"checksumSHA1": "lxe3NRr1SukdE8Mf+xNGcTUR0NI=",
+			"checksumSHA1": "4TLgSCgJZuS5gtytxNvcVk4h8/g=",
 			"path": "github.com/prometheus/common/config",
-			"revision": "167b27da48d058a9b46d84b834d67f68f0243f67",
-			"revisionTime": "2016-03-18T12:23:18Z"
+			"revision": "0866df4b85a18d652b6965be022d007cdf076822",
+			"revisionTime": "2017-06-21T12:10:01Z"
 		},
 		{
 			"checksumSHA1": "Zsc9IQzQDkOzqiAITqxEm0JXdws=",
```

Comment:
Please update all `github.com/prometheus/common/...` vendoring so everything within that repo matches.